packages/babel-cli/src/babel/dir.js

```diff
@@ -43,18 +43,29 @@ module.exports = function (commander, filenames) {
     }
   }
 
+  let  filerDotFiles = function (x) {
+    return x[0] === ".";
+  };
+
   function handle(filename) {
     if (!fs.existsSync(filename)) return;
 
     let stat = fs.statSync(filename);
 
     if (stat.isDirectory(filename)) {
       let dirname = filename;
-
-      _.each(util.readdir(dirname), function (filename) {
-        let src = path.join(dirname, filename);
-        handleFile(src, filename);
-      });
+      if (commander.copyDotFiles) {
```

Comment:
Yeah, the loop is being duplicated unnecessarily. This is what I came up with:
```
      if (commander.copyDotFiles) {
        filterUsed = filterIncludeDotFiles;
      }
      else {
        filterUsed = filterExcludeDotFiles
      }
```
I tried thinking of a way in which we can eliminate the second filter but that would require different calls to `util.readdir`. Thoughts?